fields/country_normalized.json

```diff
@@ -0,0 +1,8 @@
+{
+    "properties": {
```

Comment:
doesn't this need to be nested with the address object?

Reply:
Nice catch! How's this look?

Reply:
looks good!